src/roswire/proxy/roslaunch/config/node.py

```diff
@@ -1,20 +1,28 @@
 # -*- coding: utf-8 -*-
-__all__ = ('NodeConfig',)
+__all__ = ('NodeConfig', 'ExecutableType',)
 
 from typing import Collection, Optional, Sequence, Tuple
 import xml.etree.ElementTree as ET
+from enum import Enum
 
 import attr
 
 from ....name import namespace_join
 
 
+class ExecutableType(Enum):
+    PYTHON = 1
+    CPP = 2
```

Comment:
We might also want to change `CPP` to `LIKELY_CPP` to indicate that we have _slightly_ less confidence that a given executable was produced from C++ source code.